common/README.md

```diff
@@ -0,0 +1,60 @@
+# Opentelemetry::Common
+
+The `opentelemetry-common` gem provides common helpers for OpenTelemetry.
+
+## What is OpenTelemetry?
+
+OpenTelemetry is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+The `opentelemetry-common` gem provides common helpers for semantic conventions, context propagation patterns, etc. It depends only on the OpenTelemetry Ruby API, not the SDK.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-common
+```
+
+Or, if you use Bundler, include `opentelemetry-common` in your `Gemfile`.
+
+```rb
+require 'opentelemetry/common'
+
+# TODO: example of semantic convention helpers
+
+# Context propagation in Koala instrumentation
+OpenTelemetry::Common::HTTP::ClientContext.with_attributes('peer.service' => 'facebook') do
+  ...
```

Comment:
Could the example here simply be a `Net::HTTP` call?

Reply:
🤦 yes